src/main/java/org/numenta/nupic/network/KryoSerializer.java

```diff
@@ -25,22 +23,18 @@
  * Use instead {@link Network#serializer(SerialConfig, boolean)}
  */
 public class KryoSerializer<T> extends Serializer<T> implements Serializable {
+    private static final long serialVersionUID = 1L;
 
-    private final Class[] classes;
+    private final Class<?>[] classes;
 
     /** Use of Fast Serialize https://github.com/RuedigerMoeller/fast-serialization */
     private transient FSTConfiguration fastSerialConfig;
 
-    public KryoSerializer(Class... c) {
+    public KryoSerializer(Class<?>... c) {
         this.classes = c;
         initFST();
     }
 
-    private void readObject(ObjectInputStream in) throws IOException, ClassNotFoundException {
```

Comment:
By dumping this, you regress the functionality.   Post-deserialization the `fastSerialConfig` is null.